sessions/TypeScript-100/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Each subfolder in `100` projects is assumed as a separate project. Please move this file into the `102` folder.